src/net_processing.cpp

```diff
@@ -2958,6 +2958,59 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         return;
     }
 
+    // Received from a peer demonstrating readiness to announce transactions via reconciliations.
+    // This feature negotiation should happen: between VERSION and VERACK to avoid relay problems
+    // from switching announcement protocols after the connection is up.
+    if (msg_type == NetMsgType::SENDRECON) {
+        // For now, transaction reconciliation is an experimental feature that could be
```

Comment:
In "p2p: Finish negotiating reconciliation support"

Nit: no need to repeat this multiple times (it's already mentioned in AppInit).